watch/app/src/main/java/com/google/samples/quickstart/engagesdksamples/watch/data/converters/ItemToEntityConverter.kt

```diff
@@ -34,22 +38,37 @@ object ItemToEntityConverter {
     val movieBuilder: MovieEntity.Builder =
       MovieEntity.Builder()
         .setName(movie.movieName)
+        .setEntityId(movie.id)
         .addPosterImage(
           Image.Builder()
             .setImageUri(
               Uri.parse("android.resource://" + PACKAGE_NAME + "/" + movie.landscapePoster)
             )
             .setImageWidthInPixel(408)
             .setImageHeightInPixel(960)
+            .setImageTheme(ImageTheme.IMAGE_THEME_UNSPECIFIED)
             .build()
         )
         .setPlayBackUri(Uri.parse(movie.playbackUri))
+        .addPlatformSpecificPlaybackUri(
+          PlatformSpecificUri(Uri.parse(movie.platformSpecificPlaybackUri), movie.platformType)
+        )
         .setReleaseDateEpochMillis(movie.releaseDate)
         .setAvailability(movie.availability)
-        .setOfferPrice(movie.offerPrice)
         .setDurationMillis(movie.durationMillis)
         .addGenre(movie.genre)
-        .addContentRating(movie.contentRatings)
+        .addAvailabilityTimeWindow(
+          DisplayTimeWindow.Builder()
+            .setStartTimestampMillis(movie.startTimestampMillis)
+            .setEndTimestampMillis(movie.endTimestampMillis)
```

Comment:
Availability window refers to the timeframe for which the entity will be available for watching. Let's fill it with something other than `movie.startTimestampMillis` and `movie.endTimestampMillis` because this indicates that we can pass start and end timestamps here..

Reply:
Changed to use attributes specific to availability